### PR TITLE
Rename fields for clarity

### PR DESCRIPTION
**Pull request by eugene-chen**

### CODE REVIEW

File: src/market.rs [hunk 1]
```diff
@@ -262,8 +264,8 @@ impl<const BIDS_SIZE: usize, const ASKS_SIZE: usize, const NUM_SEATS: usize> Mar
         self.base_lots_per_base_unit
     }
 
-    fn get_quote_lots_per_tick(&self) -> u64 {
-        self.quote_lots_per_tick
+    fn get_tick_size_in_quote_lots_per_base_unit(&self) -> u64 {
```
Comment (jarry-xiao) [inline src/market.rs:267]: I hate this function name

**Review by jarry-xiao — APPROVED** [review]
Looks good. Definitely the ambiguities between units can get a little confusing. Luckily i think the phoenix-sdk conversion helper functions should mitigate this a lot

File: src/market.rs (hunk 1 above)
Comment (jarry-xiao) [reply src/market.rs:267]: Explicit, but verbose. I guess it's better than something really confusing

Comment (eugene-chen) [reply src/market.rs]: This one is removed, but I renamed `get_tick_size` to `get_tick_size_in_quote_atoms_per_base_unit`

**Pull request merged**